<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>631</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>261</width>
      <height>219</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="text">
        <string>цвет фона</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButtonFigure">
       <property name="text">
        <string>Заммкнуть фигуру</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButtonPoint">
       <property name="text">
        <string>Построить точку</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButtonNoDelay">
       <property name="text">
        <string>Без задержки</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="radioButtonDelay">
       <property name="text">
        <string>С задержкой</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>Выбор режима закраски</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBoxY"/>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="spinBoxX"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>цвета закраски</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxFon">
       <item>
        <property name="text">
         <string>Белый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxFill">
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Белый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidgetPoints">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
